README.rst

```diff
@@ -178,6 +165,7 @@ Changelog
 0.15.0 (UNRELEASED)
 ~~~~~~~~~~~~~~~~~~~
 - Add support for Python 3.9
+- Abandon support for Python 3.5
```

Comment:
Just add a note here instructing users to use a lower version if they need 3.5 support :)

Reply:
Sure! I adjusted the Changelog accordingly. Let me know what you think.